staging/src/k8s.io/cloud-provider/controllers/service/controller_test.go

```diff
@@ -55,8 +55,10 @@ import (
 
 const region = "us-central"
 
-func newService(name string, uid types.UID, serviceType v1.ServiceType) *v1.Service {
-	return &v1.Service{
+type serviceTweak func(s *v1.Service)
+
+func newService(name string, uid types.UID, serviceType v1.ServiceType, tweaks ...serviceTweak) *v1.Service {
```

Comment:
I don't think the UID argument is ever useful and just makes tests look noisy - can we get rid of it?  Just set it to a default value in here.

Reply:
sure